[DOCS] OtherTests.generateFunctionIndex fails for UNNEST WITH ORDINALITY with Could Not Find Function in <file> error

OtherTests.generateFunctionIndex fails to detect `UNNEST WITH ORDINALITY` even though it is used in the Python tests.

It searches for an exact match: `UNNEST WITH ORDINALITY`  
But the syntax is: `UNNEST(<expr>) WITH ORDINALITY`.

Running this test after adding the string containing Python test file names for function in `CalciteFunctions.java` fails with the following error:
```
[ERROR]   OtherTests.generateFunctionIndex:758 
� Runtime Could not find function UNNEST WITH ORDINALITY in ../../python/tests/runtime_aggtest/complex_type_tests/test_arr.py
[12:46 AM]
```

--- a/sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/calciteCompiler/CalciteFunctions.java
+++ b/sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/calciteCompiler/CalciteFunctions.java
@@ -202,13 +202,11 @@ public class CalciteFunctions implements FunctionDocumentation.FunctionRegistry 
             new Func(SqlStdOperatorTable.IGNORE_NULLS, "IGNORE NULLS", SqlLibrary.STANDARD, "grammar#window-aggregates", FunctionDocumentation.NO_FILE, false),
             new Func(SqlStdOperatorTable.RESPECT_NULLS, "RESPECT NULLS", SqlLibrary.STANDARD, "grammar#window-aggregates", FunctionDocumentation.NO_FILE, false),
             new Func(SqlStdOperatorTable.MINUS_DATE, "-", SqlLibrary.STANDARD, "datetime", FunctionDocumentation.NO_FILE, false),
-            new Func(SqlStdOperatorTable.UNNEST, "UNNEST", SqlLibrary.STANDARD, "array#the-unnest-sql-operator,map#the-unnest-operator", FunctionDocumentation.NO_FILE, false),
-            new Func(SqlStdOperatorTable.UNNEST_WITH_ORDINALITY, "UNNEST WITH ORDINALITY", SqlLibrary.STANDARD, "", FunctionDocumentation.NO_FILE,
-//                 """
-//                 runtime_aggtest/complex_type_tests/test_{arr,arr_unnest,arr_arr_unnest,arr_row_unnest,row_arr_unnest,arr_map_unnest,arr_udt_unnest,arr_of_arr,arr_of_row,arr_of_map,arr_of_udt,row_of_arr,udt_of_arr,map_var_cmpx}.py|
-//                 runtime_aggtest/variant_tests/{row_of_cmpx_type,cpmx_variant,arr_cmpx_varnt_unnest,arr_of_cmpx_type,udt_of_cmpx_type,arr_unnest_varnt}.py
-//                 """,
-            false),
+            new Func(SqlStdOperatorTable.UNNEST, "UNNEST", SqlLibrary.STANDARD, "array#the-unnest-sql-operator,map#the-unnest-operator", """
+                 runtime_aggtest/complex_type_tests/test_{arr,arr_unnest,arr_arr_unnest,arr_row_unnest,row_arr_unnest,arr_map_unnest,arr_udt_unnest,arr_of_arr,arr_of_row,arr_of_map,arr_of_udt,row_of_arr,udt_of_arr,map_var_cmpx}.py|
+                 runtime_aggtest/variant_tests/{row_of_cmpx_type,cpmx_variant,arr_cmpx_varnt_unnest,arr_of_cmpx_type,udt_of_cmpx_type,arr_unnest_varnt}.py""", false),
+            // Unnest with ordinality is the same as UNNEST
+            new Func(SqlStdOperatorTable.UNNEST_WITH_ORDINALITY, "UNNEST WITH ORDINALITY", SqlLibrary.STANDARD, "", FunctionDocumentation.NO_FILE, false),
             new Func(SqlStdOperatorTable.LATERAL, "LATERAL", SqlLibrary.STANDARD, "grammar#lateral", FunctionDocumentation.NO_FILE, false),
             new Func(SqlStdOperatorTable.COLLECTION_TABLE, "TABLE", SqlLibrary.STANDARD, "grammar", FunctionDocumentation.NO_FILE, false),
 

--- a/sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/calciteCompiler/FunctionDocumentation.java
+++ b/sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/frontend/calciteCompiler/FunctionDocumentation.java
@@ -143,6 +143,8 @@ public class FunctionDocumentation {
                 File file = path.toFile();
                 if (!file.exists())
                     throw new RuntimeException("File " + file.getPath() + " does not exist");
+                if (!file.isFile())
+                    throw new RuntimeException("File " + file.getPath() + " is not a file");
                 String contents = fileContents.getContents(file.getPath());
                 if (!description.functionName().isEmpty() &&
                         !contents.toLowerCase().contains(description.functionName().toLowerCase()))

--- a/sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/util/Utilities.java
+++ b/sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/util/Utilities.java
@@ -326,6 +326,7 @@ public class Utilities {
     }
 
     public static String readFile(Path filename) throws IOException {
+        Utilities.enforce(!filename.toFile().isDirectory(), () -> filename + " is a directory");
         List<String> lines = Files.readAllLines(filename);
         return String.join(System.lineSeparator(), lines);
     }

